### PR TITLE
Query log setup

### PR DESCRIPTION
**Pull request by SeBBBe**
* Add query log params to service examples
* Clarify Neo4j 5+ requirement for query logs
* Cross-reference query log configuration in feature overview page

----

If you open a PR that needs to go into a current version, you need to *cherry-pick your commit from dev over to the current version branch*. Only then will the proper builds that generate html/pdf be run. But beware: Docs will be generated but not published automatically!

- [x] N/A - or - I have added the appropriate "cherry-pick-to" labels to this PR so I don't forget to do this later!

### CODE REVIEW

**Review by eve-bright — APPROVED** [review]

File: modules/ROOT/pages/addition/index.adoc [hunk 1]
```diff
@@ -240,7 +241,7 @@ systemctl edit neo4j-ops-manager-agent.service
 ----
 +
 and set environment variables by either setting Environment or EnvironmentFile options.
-For example, using the Environment options, the override file looks as follows:
+For example, using the Environment options, the override file can look as follows:
```
Comment (AlexicaWright) [inline modules/ROOT/pages/addition/index.adoc]:
```suggestion
For example, using the Environment options, the override file can look like this:
```

File: modules/ROOT/pages/addition/instance-requirements.adoc [hunk 2]
```diff
@@ -1,18 +1,29 @@
...
 * be running a Neo4j Enterprise Edition v4.4.0 or higher (including all 5.x versions).
 
+== Query log collection
+
+[NOTE]
+====
+
+Log manager is only compatible with Neo4j version 5+.
+
+====
```
Comment (AlexicaWright) [inline modules/ROOT/pages/addition/instance-requirements.adoc]:
```suggestion
====
Log manager is only compatible with Neo4j version 5+.
====
```

File: modules/ROOT/pages/addition/instance-requirements.adoc [hunk 3]
```diff
@@ -39,3 +50,28 @@ For the metrics collection feature to work correctly. these configuration values
 ** `server.metrics.jmx.enabled=true`
 
 ** `server.metrics.namespaces.enabled=true`
+
+== User privileges
```
Comment (AlexicaWright) [inline modules/ROOT/pages/addition/instance-requirements.adoc:52]:
```suggestion
== User privileges

```

File: modules/ROOT/pages/addition/instance-requirements.adoc [hunk 4]
```diff
@@ -39,3 +50,28 @@ For the metrics collection feature to work correctly. these configuration values
...
+The agent logs on to the DBMS with the configured user to enable certain features such as viewing relationship and label types, managing privileges and viewing configuration values.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/addition/instance-requirements.adoc]:
```suggestion
The agent logs on to the DBMS with the configured user to enable certain features such as viewing relationship and label types, managing privileges, and viewing configuration values.
```

File: modules/ROOT/pages/monitoring/log-manager.adoc [hunk 5]
```diff
@@ -4,6 +4,13 @@
 The log manager allows you to filter, sort and browse query logs from each of your monitored DBMS instances. 
 At first, an aggregated **Summary** view of queries performed within the last 15 minutes is displayed.
 
+[NOTE]
+====
+
+If you are not seeing any log messages, or you see a message saying that logs are not enabled, please refer to *xref:../addition/instance-requirements.adoc[Neo4j instance requirements]* and *xref:../addition/index.adoc#querylog[Configure agent]*.
+
+====
```
Comment (AlexicaWright) [inline modules/ROOT/pages/monitoring/log-manager.adoc]:
```suggestion
====
If you are not seeing any log messages, or if you see a message saying that logs are not enabled, please refer to *xref:../addition/instance-requirements.adoc[Neo4j instance requirements]* and *xref:../addition/index.adoc#querylog[Configure agent]*.
====
```

**Review by AlexicaWright — COMMENTED** [review]
Looks good, I left a few editorial suggestions.

File: modules/ROOT/pages/addition/instance-requirements.adoc (hunk 4 above)
Comment (SeBBBe) [reply modules/ROOT/pages/addition/instance-requirements.adoc]: Oxford comma! 😱

**Pull request merged**